packages/browser/__tests__/login/oidc/oidcHandlers/AuthorizationCodeWithPkceOidcHandler.spec.ts

```diff
@@ -129,7 +133,27 @@ describe("AuthorizationCodeWithPkceOidcHandler", () => {
       );
     });
 
+    it("supports silent authentication", async () => {
```

Comment:
Seems this test is more specifically about setting an option and asserting that the low-level library is called with it
```suggestion
    it("passes our the 'prompt' option down to our OIDC client library implementation", async () => {
```